panacea/grpc_client.go

```diff
@@ -63,7 +78,41 @@ func (c *GrpcClient) GetPubKey(panaceaAddr string) ([]byte, error) {
 	if err := c.interfaceRegistry.UnpackAny(response.GetAccount(), &acc); err != nil {
 		return nil, fmt.Errorf("failed to unpack account info: %w", err)
 	}
-	return acc.GetPubKey().Bytes(), nil
+	return acc.GetPubKey(), nil
+}
+
+func (c *GrpcClient) GetAccountNumber(panaceaAddr string) (uint64, error) {
```

Comment:
I think you can make one function `GetAccountNumberAndSequence()` or `GetAccount()` because you always need both account number and sequence anyway.

Reply:
I agree with your review. I'll refactor it.